Source/Plugins/Core/com.equella.core/js/tsrc/util/langstrings.ts

```diff
@@ -301,6 +301,8 @@ export const languageStrings = {
       done: "Done",
       yes: "Yes",
       no: "No",
+      showmore: "Show more",
+      showless: "Show less",
```

Comment:
It was agreed we're going to aim for camelCase here, so:

```suggestion
      showMore: "Show more",
      showLess: "Show less",
```